cfn/README-GD.md

```diff
@@ -59,125 +67,110 @@ An example of a successful response is:
 1. Select the region in which you want to deploy the CFT.
-1. Click `Services`->`CloudFormation`->`Create Stack`.
-1. In the `Choose a template` section select `Specify an Amazon S3 template URL`, use the following URL and click `Next`:
+1. Click **Services**->**CloudFormation**->**Create Stack**.
+1. In the `Choose a template` section select **Specify an Amazon S3 template URL** and enter the following URL.
 `https://s3.amazonaws.com/alertlogic-collectors-us-east-1/cfn/guardduty.template`
+1. Click **Next**.
 1. On the `Specify Details` window, provide the following required parameters:
    - `Stack name` - Any name you have used for creating an AWS stack
    - `AccessKeyId` - `access_key_id` returned from AIMs [above](#alert-logic-access-key-creation)
    - `AlApiEndpoint` - usually `api.global-services.global.alertlogic.com` 
    - `AlDataResidency` - usually `default`
    - `SecretKey` - `secret_key` returned from AIMs [above](#alert-logic-access-key-creation)
-1. Click Next. 
-1. On the Options panel, click Next.
-1. In the Review panel, perform a predeployment check. 
-1. Select "I acknowledge that AWS CloudFormation might create IAM resources," and then click Create.
-1. On the CloudFormation, Stacks panel, filter based on the stack name you created, and then 
+1. Click **Next**. 
+1. On the `Options` panel, click **Next**.
+1. In the `Review` panel, perform a predeployment check. 
+1. Select **"I acknowledge that AWS CloudFormation might create IAM resources"**, then click **Create**.
+1. On the CloudFormation, `Stacks` panel, filter based on the stack name you created, and then 
 select your stack by name.
 
+If the deployment was successful, the status appears as: CREATE_COMPLETE. If the deployment was not successful, 
+see [Troubleshooting Installation Issues](#troubleshooting-installation-issues) below.
+
 **Note:** Only one collector installation is allowed per AWS region. If you try to deploy the template multiple times in the same region, if will fail with the following error:
 
 Status | Type | Logical ID | Status Reason
 --|--|--|--
 CREATE_FAILED | AWS::Lambda::Function | CollectLambdaFunction | alertlogic-cwe-collector already exists in stack arn:aws:cloudformation:us-east-1:123456789101:stack/test-one/f9536300-d12b-11e7-ac98-50d5cd16c68e
 
 
-If the deployment was successful, the status appears as: CREATE_COMPLETE. If the deployment was not successful, 
-see [Troubleshooting Installation Issues](#troubleshooting-installation-issues) below.
-
-#### Use a Command Line to deploy
+### Use a Command Line to deploy
 
-Follow these steps to deploy the Alert Logic custom template using the [AWS CLI](https://aws.amazon.com/cli/).
+Follow these steps to deploy the Alert Logic custom template using the [AWS CLI](https://aws.amazon.com/cli/). This procedure requires the previously downloaded [Alert Logic custom CFT](https://github.com/alertlogic/cwe-collector/blob/master/cfn/guardduty.template).
```

Comment:
Please remove the sentence "This procedure requires the previously downloaded" , it turns out this step is not needed at all (since we specify url to custom CFT as parameter for `aws cloudformation create-stack` command)